app/src/main/java/com/vuongnq14798/vuongmp3/data/source/TracksDataSource.kt

```diff
@@ -1,8 +1,20 @@
 package com.vuongnq14798.vuongmp3.data.source
 
+import com.vuongnq14798.vuongmp3.data.model.Track
+
 interface TracksDataSource {
 
-    interface Remote {}
+    interface LoadTracksCallback {
```

Comment:
ở đây ông có thể sử dụng generic type để tăng tính tái sử dụng của code: 
```suggestion
    interface LoadDataCallback<T> {
```